webapp/login/views.py

```diff
@@ -123,113 +125,6 @@ def after_login(resp):
     return response
 
 
-@login.route("/login-beta", methods=["GET"])
```

Comment:
Can we add a redirect to `/login` just incase someone's last login method was “beta”?

Reply:
Ok